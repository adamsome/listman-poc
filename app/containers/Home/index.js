import React from 'react'

const Home = () => (
  <div className="home">
    Under Construction
  </div>
)

export default Home
