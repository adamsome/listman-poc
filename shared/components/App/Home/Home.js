import React from 'react';
import Helmet from 'react-helmet';
import getConfig from '../../../../config/get';

import BodyColumn from '../../Layout/BodyColumn'

const Home = () => (
  <BodyColumn>
    <Helmet title="Home" />

    <h2 className="title is-1">{getConfig('welcomeMessage')}</h2>

    <p className="subtitle is-5">
      Under Construction!
    </p>
  </BodyColumn>
)

export default Home;
