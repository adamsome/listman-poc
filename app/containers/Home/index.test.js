import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Home from './index'

it('renders correctly', () => {
  const wrapper = shallow(<Home />)
  expect(toJson(wrapper)).toMatchSnapshot()
})

