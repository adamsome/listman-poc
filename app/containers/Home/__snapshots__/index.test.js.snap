exports[`test renders correctly 1`] = `
<div
  className="home">
  Under Construction
</div>
`;
